src/mpl2/README.md

```diff
@@ -65,7 +65,7 @@ rtl_macro_placer
 | `-target_dead_space` | Specifies the target dead space percentage, which influences the utilization of `StandardCellCluster`. The allowed values are floats, and the default value is `0.05`. |
 | `-min_ar` | Specifies the minimum aspect ratio $a$, or the ratio of its width to height of a `StandardCellCluster` from $[a, \frac{1}{a}]$. The allowed values are floats, and the default value is `0.33`. |
 | `-snap_layer` | Snap macro origins to this routing layer track. The default value is 4, and the allowed values are integers `[1, MAX_LAYER]`). |
-| `-bus_planning_flag` | Flag to enable bus planning. The recommendation is to turn on bus planning for SKY130 and off for NanGate45/ASAP7. The default value is disabled. |
+| `-bus_planning` | Flag to enable bus planning. The recommendation is to turn on bus planning for SKY130 and off for NanGate45/ASAP7. The default value is disabled. |
```

Comment:
This appears to be based on https://github.com/The-OpenROAD-Project/OpenROAD/blob/e4ca4c0b0bbf5fe69cb4e6083f4793e9d2a2ffca/src/mpl2/src/hier_rtlmp.h#L272

I think the description about limited routing layers is more helpful as people may use this with other PDKs.  GF180 sounds like another candidate for bus planning.

I know this is incidental to your change but let's fix it while we are looking at it.

Reply:
Should I leave these technologies used as examples or just use the description about limited routing layers?